laCour/slack-night-mode
# lighten the status text shown on a user menu

Resolves #130

This commit uses $color-highlight for the status text shown next to a
user's status emoji under their name and avatar in the popup menu.

Test Plan:
- Set your user's status to something that includes text (not emoji only)
- Click on your name in any channel to trigger your profile popup
- Verify that the status text (just to the right of the emoji is light
  enough to easily read)
